components/ui/tab.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tab as HeadlessTab } from "@headlessui/react";
+import type { ExtractProps } from "../types";
+
+export const Tab = (props: ExtractProps<typeof HeadlessTab>) => {
+  return (
+    <HeadlessTab
+      className="aria-selected:bg-white aria-selected:text-gray-700 aria-selected:font-medium aria-selected:rounded-md aria-selected:shadow px-3 py-2 text-gray-500 outline-none"
```

Comment:
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/11313640/227560908-7696e472-6590-4b79-be12-7c4da4b82f7a.png">


Here's image for reference - few things to update:
 - margin is missing (`m-1.5`),
 - padding is not the same as in mocks (`px-3.5 py-2.5`),
 - font weight is 600 for both state (default and selected) (`text-semibold`)

Reply:
Here's code with classes I've mentioned if you want to apply changes right away
```suggestion
      className="m-1.5 px-3.5 py-2.5 text-gray-500 font-semibold aria-selected:bg-white aria-selected:text-gray-700 aria-selected:rounded-md aria-selected:shadow focus:outline-none"
```

Reply:
Regarding the padding, I was probably looking at different Tab component, because I remember I saw 12px and 8px padding, but this is fine.

Also I was experimenting with font weight :)

This looks good after changes 👍

Reply:
Yeah, it might be my mistake for not providing you the link to the tab component with right size (so far we've defaulted to `md` size for table, button and checkboxes).